debian/vitrobian-crystal.yaml

```diff
@@ -13,20 +13,29 @@ actions:
     suite: "stretch"
     components:
       - main
+      - contrib
+      - non-free
     mirror: https://deb.debian.org/debian
-    variant: minbase
 
   - action: apt
     description: Install base packages
     packages: [ sudo, openssh-server, adduser, systemd-sysv ]
 
   - action: apt
     description: Install extended packages
-    packages: [ locales, ssh, less, fbset, psmisc, strace, ed, ncdu, crda, console-setup, keyboard-configuration, debconf-utils, parted, unzip, build-essential, manpages-dev, python, bash-completion, gdb, pkg-config, avahi-daemon, hardlink, ca-certificates, curl, nfs-common, usbutils, dosfstools, apt-listchanges, usb-modeswitch, apt-transport-https, libmtp-runtime, rsync, htop ]
+    packages: [ locales, ssh, less, fbset, psmisc, strace, ed, ncdu, crda, console-setup, keyboard-configuration, debconf-utils, parted, unzip, build-essential, manpages-dev, python, bash-completion, gdb, pkg-config, avahi-daemon, hardlink, ca-certificates, curl, nfs-common, usbutils, dosfstools, apt-listchanges, usb-modeswitch, apt-transport-https, libmtp-runtime, rsync, htop, nano ]
 
   - action: apt
     description: Install network packages
-    packages: [ wpasupplicant, wireless-tools, dhcpcd5, net-tools, rfkill ]
+    packages: [ wpasupplicant, wireless-tools, dhcpcd5, net-tools, rfkill, iputils-ping, wget ]
+
+  - action: apt
+    description: Install packages needed to run greengrass core
+    packages: [ sqlite3 ]
+
+  - action: apt
+    description: Extended packages from Raspbian
+    packages: [ netbase, v4l-utils, lua5.1, luajit, fake-hwclock, libfreetype6-dev, dphys-swapfile, man-db, policykit-1, ssh-import-id, firmware-atheros, firmware-brcm80211, firmware-libertas, firmware-misc-nonfree, firmware-realtek ]
```

Comment:
@lukasz-laguna We can include those in the ` Install extended packages` section.